### ISSUE TITLE
Wrong date format when working in a non EN environment

### ISSUE THREAD

**Issue by LoOnGitH** [MarkZither/KimaiDotNet#17]
![image](https://user-images.githubusercontent.com/4036743/123414436-2c5c2d00-d5b4-11eb-811f-c872a19688dc.png)

Format is dd-mmm-yyyy but it seems not understandable in FR (should be jj-mmm-aaaa).

**Comment by MarkZither** [comment]
It is currently set to `cell.NumberFormat = "dd-MMM-yyyy";` which is fine for my English excel...

following this suggestion https://stackoverflow.com/a/35394640 i have tried to make it a bit more friendly in the next version

``` c#
var yearCode = Globals.ThisAddIn.Application.International[XlApplicationInternational.xlYearCode];
var monthCode = Globals.ThisAddIn.Application.International[XlApplicationInternational.xlMonthCode];
var dayCode = Globals.ThisAddIn.Application.International[XlApplicationInternational.xlDayCode];
// https://stackoverflow.com/questions/3310800/how-to-make-correct-date-format-when-writing-data-to-excel
var cell = Worksheet.Range[Worksheet.Cells[2, ExcelAddin.Constants.Sheet1.DateColumnIndex], Worksheet.Cells[10000, ExcelAddin.Constants.Sheet1.DateColumnIndex]];
cell.NumberFormat = string.Format("{0}{1}-{2}{3}-{4}{5}{6}{7}", dayCode, dayCode, monthCode, monthCode, yearCode, yearCode, yearCode, yearCode); // e.g. dd-MMM-yyyy
var cellBegin = Worksheet.Range[Worksheet.Cells[2, ExcelAddin.Constants.Sheet1.BeginTimeIndex], Worksheet.Cells[10000, ExcelAddin.Constants.Sheet1.BeginTimeIndex]];
```

**Comment by MarkZither** [comment]
if this works i will extend it to cover time too
``` c#
var cellBegin = Worksheet.Range[Worksheet.Cells[2, ExcelAddin.Constants.Sheet1.BeginTimeIndex], Worksheet.Cells[10000, ExcelAddin.Constants.Sheet1.BeginTimeIndex]];
            cellBegin.NumberFormat = "hh:mm:ss"; // e.g. dd-MMM-yyyy
            var cellEnd = Worksheet.Range[Worksheet.Cells[2, ExcelAddin.Constants.Sheet1.EndTimeIndex], Worksheet.Cells[10000, ExcelAddin.Constants.Sheet1.EndTimeIndex]];
            cellEnd.NumberFormat = "hh:mm:ss"; // e.g. dd-MMM-yyyy
```

**Comment by MarkZither** [comment]
merged to main with v0.2
